cmd/apps/helm_operator_app.go

```diff
@@ -0,0 +1,141 @@
+// Copyright (c) arkade author(s) 2020. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+package apps
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"path"
+
+	"github.com/alexellis/arkade/pkg/apps"
+	"github.com/alexellis/arkade/pkg/config"
+	"github.com/alexellis/arkade/pkg/env"
+	"github.com/alexellis/arkade/pkg/helm"
+	"github.com/alexellis/arkade/pkg/k8s"
+	"github.com/alexellis/arkade/pkg/types"
+	"github.com/spf13/cobra"
+	"golang.org/x/mod/semver"
+)
+
+func MakeInstallHelmOperator() *cobra.Command {
+	var helmOperator = &cobra.Command{
+		Use:          "helm-operator",
+		Short:        "Install helm-operator",
+		Long:         "Install helm-operator",
+		Example:      "arkade install helm-operator --namespace default",
+		SilenceUsage: true,
+	}
+
+	helmOperator.Flags().StringP("namespace", "n", "default", "The namespace used for installation")
+	helmOperator.Flags().Bool("update-repo", true, "Update the helm repo")
+	helmOperator.Flags().Bool("helm3", true, "Use helm3, if set to false uses helm2")
+	helmOperator.Flags().StringP("version", "v", "v1.2.0", "The version of helm-operator to install")
+
+	helmOperator.RunE = func(command *cobra.Command, args []string) error {
+		wait, _ := command.Flags().GetBool("wait")
+
+		namespace, _ := command.Flags().GetString("namespace")
+		version, _ := command.Flags().GetString("version")
+		helm3, _ := command.Flags().GetBool("helm3")
+		updateRepo, _ := helmOperator.Flags().GetBool("update-repo")
+
+		if !semver.IsValid(version) {
+			return fmt.Errorf("%q is not a valid semver version", version)
+		}
+
+		if namespace != "default" {
+			return fmt.Errorf(`to override the "default", install via tiller`)
+		}
+
+		userPath, err := config.InitUserDir()
+		if err != nil {
+			return err
+		}
+
+		clientArch, clientOS := env.GetClientArch()
+
+		fmt.Printf("Client: %s, %s\n", clientArch, clientOS)
+
+		log.Printf("User dir established as: %s\n", userPath)
+
+		os.Setenv("HELM_HOME", path.Join(userPath, ".helm"))
+
+		_, err = helm.TryDownloadHelm(userPath, clientArch, clientOS)
+		if err != nil {
+			return err
+		}
+
+		helmRepo := "fluxcd/helm-operator"
+		if !helm3 {
+			helmRepo = "helm-operator/rendered"
+		}
+
+		err = helm.FetchChart(helmRepo, defaultVersion)
+		if err != nil {
+			return err
+		}
+
+		overrides := map[string]string{}
+		overrides["helm.versions"] = "v3"
+
+		arch := k8s.GetNodeArchitecture()
+		if arch != "amd64" {
+			return fmt.Errorf("This chart does not support %s", arch)
+		}
+
+		installOptions := types.DefaultInstallOptions().
+			WithNamespace(namespace).
+			WithHelmPath(path.Join(userPath, ".helm")).
+			WithHelmRepo(helmRepo).
+			WithHelmURL("https://charts.fluxcd.io").
+			WithOverrides(overrides).
+			WithWait(wait).
+			WithHelmUpdateRepo(updateRepo)
+
+		if command.Flags().Changed("kubeconfig") {
```

Comment:
We should always be getting and setting the kubeconfig flag value and setting it in the options, not only when its changed. 

always grab the value at the start
```go
kubeConfigPath, _ := command.Flags().GetString("kubeconfig")
```
then add the below to the optionas

```go 
			WithKubeconfigPath(kubeConfigPath)

```

Reply:
Done